.rubocop.yml

```diff
@@ -66,6 +67,18 @@ Metrics/PerceivedComplexity:
 
 Naming/AccessorMethodName:
   Enabled: false
+Naming/UncommunicativeBlockParamName:
+  AllowedNames:
```

Comment:
These are just examples of violations from AgBiome that seemed reasonable (`x` and `y` referring to Cartesian coordinates).